contrib/test_jenkins.sh

```diff
@@ -939,6 +939,31 @@ test_jucx() {
 		jucx_port=$((20000 + EXECUTOR_NUMBER))
 		export JUCX_TEST_PORT=$jucx_port
 		$MAKE -C bindings/java/src/main/native test
+	        ifaces=`ibdev2netdev | grep Up | awk '{print $5}'`
+		if [ -n "$ifaces" ]
+		then
+                        $MAKE -C bindings/java/src/main/native package
+		fi
+		for iface in $ifaces 
+		do
+			if [ -n "$iface" ]
+                	then
+                   		server_ip=`ip addr show ${iface} | awk '/inet /{print $2}' | awk -F '/' '{print $1}'`
+                	fi
+
+                	if [ -z "$server_ip" ]
+                	then
+		   	   	echo "Interface $iface has no IPv4"	
+                   	   	continue
+                        fi
+                        echo "Running standalone benchamrk on $iface"
+
+                        java -cp bindings/java/src/main/native/build-java/jucx-*.jar org.ucx.jucx.examples.UcxReadBWBenchmarkReceiver s=$server_ip p=$JUCX_TEST_PORT 2>&1 > bindings/java/src/main/native/build-java/jucx_bench.log & 
+                        sleep 10
+                        java -cp bindings/java/src/main/native/build-java/jucx-*.jar  org.ucx.jucx.examples.UcxReadBWBenchmarkSender s=$server_ip p=$JUCX_TEST_PORT t=10000000
+                        cat bindings/java/src/main/native/build-java/jucx_bench.log
```

Comment:
why need to redirect to a file? if 1st command fails, we will not see the output in jenkins console..

Reply:
But 1 command is blocking. It' waiting for connections.

Reply:
it can run in background and still continue to print to stdout

Reply:
if many iterations it'll mix output with the second command. Ok, will disable file piping.